File: f987(hello_world_2038)/C++/hello_world.cpp
#include <chrono>
#include <ctime>
#include <iostream>
#include <thread>


int main(int argc, char const *argv[]) {

  switch (std::time(nullptr) % 10) {
    case 0:
    case 1:
      std::cout << "Hello, World!" << '\n';
      break;
    case 2:
    case 3:
      std::cout << "hello, world" << '\n';
      break;
    case 4:
    case 5:
      std::cout << "Halo, word!!!" << '\n';
      break;
    case 6:
    case 7:
      std::cout << "世界，您好！" << '\n';
      break;
    case 8:
    case 9:
      std::cout << "https://zerojudge.tw/ShowProblem?problemid=a001" << '\n';
      break;
    default:
      break;
  }
  std::this_thread::sleep_for(std::chrono::seconds(2));

  return 0;
}

File: f987(hello_world_2038)/Python/hello_world.py
import time


def main() -> None:
    t = int(time.time()) % 10

    if t == 0 or t == 1:
        print('Hello, World!')
    elif t == 2 or t == 3:
        print('hello, world')
    elif t == 4 or t == 5:
        print('Halo, word!!!')
    elif t == 6 or t == 7:
        print('世界，您好！')
    elif t == 8 or t == 9:
        print('https://zerojudge.tw/ShowProblem?problemid=a001')
    else:
        print(t, 'Try again.')


if __name__ == '__main__':
    main()
    time.sleep(2)
